v21.2/known-limitations.md

```diff
@@ -560,12 +513,6 @@ $ export COCKROACH_SQL_CLI_HISTORY=.cockroachsql_history_shell_2
 
 [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/42027)
 
-### Passwords with special characters cannot be passed in connection parameter
```

Comment:
@knz, please confirm that it's ok to remove this limitation from the 21.2 docs.

Reply:
The limitation is transformed and will  need a doc change: if the password contains special characters, it should be passed as a query argument on the right.